README.md

```diff
@@ -18,4 +18,4 @@ docker run --rm \
     openmaptiles/import-water
 ```
 ## Version of OpenStreetMapData
-**2019-02-21**
+**2019-05-13**
```

Comment:
@eva-j are you going to upload an image you've created with data from 2019-05-13 to dockerhub?

Just a note: 
There is a `0.7` realease available but no image tagged `0.7 `on dockerhub: https://hub.docker.com/r/openmaptiles/import-water/tags
https://github.com/openmaptiles/import-water/releases

Reply:
I am asking @MartinMikita for help with this - to make the version and tag numbering clear.
If there is new data on https://osmdata.openstreetmap.de/data/water-polygons.html in time of making this release, we will use the most up to date version for docker image.